app/src/main/java/com/example/dungeontest/graph/GraphParser.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.dungeontest.graph
+
+import okhttp3.HttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.jgrapht.Graph
+import org.jgrapht.graph.DefaultEdge
+import org.jgrapht.nio.dot.DOTExporter
+import java.io.StringWriter
+
+class GraphParser {
+//    fun graphToPng(graph: Graph<MapRoom, DefaultEdge>): File {
+//        val dot = graphToDot(graph)
+//        return dotToPng(dot)
+//    }
+    private fun graphToDot(graph: Graph<MapRoom, DefaultEdge>): String {
+        val exporter = DOTExporter<MapRoom, DefaultEdge>()
+        val writer = StringWriter()
+
+        return exporter.exportGraph(graph, writer).toString()
```

Comment:
Just a curiosity thing, but I'm so confused. Why is this compiling!?! The source code for DotExporter.exportGraph() says that method returns void. How/why can you/we call toString() on the result of a void method?! :(